Purchases/Public/SPMSwiftInterfaces/RCTransaction.h

```diff
@@ -0,0 +1,28 @@
+//
+//  RCTransaction.h
+//  Purchases
+//
+//  Created by Andrés Boedo on 8/13/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(Purchases.Transaction)
+@interface RCTransaction : NSObject
+
+@property (nonatomic, readonly, copy) NSString *revenueCatId;
```

Comment:
one easy way to generate the code for this file, so that you don't have to manually do the conversion into objc, is to make the file public in swift, then go to derived data and look for `Purchases-swift.h`. That file is an objc version of the public interface for the framework, you just have to find the class and copy/paste it. then maybe clean it up a bit.